### ISSUE TITLE
Posts containing code tags are formatted with unwanted markup.

### ISSUE THREAD

**Issue by mpsmc** [r04r/Minichan#3]
When making a post containing code tags, the wiki markup interferes with the formatting.
```
__________________________________
/ This means that cowsay posts are \\
\ formatted incorrectly.           /
 ----------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\\/\\
                ||----w |
                ||     ||
```

There are three issues with the `parse` function:

1. The contents of the code tags are marked up. In the example above, underscores will be interpreted as underlines.
2. The newlines are substituted with `<br/>` tags, despite being contained in a `<pre>` element.
3. The CSS for `.codebox` in `layout.css` specifies `whitespace: normal`.

It would be good if the ##[code]...[/code]## markup was able to cope with this.

**Comment by r04r** [comment]
This should be fixed since bb2a54e618f5171c9e3d20ebdef7a3485cedb0ec

**Comment by mpsmc** [comment]
``` 
____________
< Good stuff >
 ------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```
